common-changenotifier/src/main/java/org/duracloud/common/changenotifier/SnsSubscriptionManager.java

```diff
@@ -72,14 +71,13 @@ public synchronized void connect() {
         attributes.put("ReceiveMessageWaitTimeSeconds", "20");
         request.setAttributes(attributes);
         CreateQueueResult result;
-        try {
+
+        this.queueUrl = getQueueUrl(this.queueName);
+        if (this.queueUrl == null) {
             result = sqsClient.createQueue(request);
```

Comment:
Does this call try to recreate a queue if one already exists? The DURACLOUD-1299 ticket suggests that the failure occurs because we're trying to delete then recreate a queue too quickly, but I'm not seeing anything here (in the original code) that tries to delete. So I'm assuming this call just fails if the queue already exists? If so, the changes in this PR fix that problem. If not, is there another code location where we're doing a delete?